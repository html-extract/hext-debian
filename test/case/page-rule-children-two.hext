<div class:class>
  <span>
    <img />
  </span>
</div>
