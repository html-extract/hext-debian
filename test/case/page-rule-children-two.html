<div class="yes1">
  <span>
    <img/>
  </span>
</div>
<div class="yes2">
  <span>
    <img/>
  </span>
  <span>
    <img/>
  </span>
</div>
<div class="no1">
</div>
<div class="no2">
  <span></span>
</div>
<div class="no3">
  <span>
    <span><img/></span>
  </span>
</div>
<div class="no4">
  <a>
    <img/>
  </a>
</div>
<div class="no5">
  <span>
    <span></span>
  </span>
</div>
