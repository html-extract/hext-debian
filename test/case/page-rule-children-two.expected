{"class":"yes1"}
{"class":"yes2"}
